packages/ui-toolkit/src/components/atoms/Toaster/assets.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
```

Comment:
Can create a `utils` folder and move `assets.tsx` and `hooks.tsx` within it.

Reply:
done